appcelerator-archive/amp
# [Monitoring] Capture the container labels in monitoring and forward to ES

Closes #956 

Collect all containers labels (including user ones) and make them part of logs / metrics metadata

## how to test

-  rebuild all including cli
- hack/deploy
- see in Kibana all the labels in logs and metrics prefixed by labels.
